src/app/view/Lock/useLockScreen.ts

```diff
@@ -111,6 +112,13 @@ export const useLockScreenProps = (props: LockScreenProps): LockViewProps => {
     [biometryEnabled]
   )
 
+  // The HomeView should have called hideSplashScreen() already,
+  // but in case it didn't, we do it here as a fallback as it is critical.
```

Comment:
I think those case are when a cozy-app is awaited by the home (cozyAppFallback), in that case no hideSplashScreen is called until the cozy-app is rendered. And because you stopped the navigation to the cozy-app to display the secure prompt, then you have to do it by yourself

Reply:
Ok, then these calls seem needed